app/services/pending_payments.rb

```diff
@@ -0,0 +1,15 @@
+# Returns the capturable payment object for an order with balance due
+
+class PendingPayments
+  def initialize(order)
+    @order = order
+  end
+
+  def payment_object
+    @order.payments.find{ |payment| payment.state == 'checkout' }
+  end
+
+  def can_be_captured?
+    payment_object && payment_object.actions.include?('capture')
```

Comment:
I just noticed that you call payment_object twice. It's not a big deal, but it will search through all payments twice as well. This should call it only once:
```ruby
payment_object.andand.actions.include?('capture')
```